product-service/helper.js

```diff
@@ -20,12 +20,24 @@ module.exports.notFoundResponse = item => {
     const message = item + ' not found';
     return response({
         message
-    }, 400);
+    }, 404);
 }
 
 module.exports.unexpectedErrorResponse = () => {
     const message = 'Unexpected error has occurred';
     return response({
         message
     }, 500);
+}
+
+module.exports.validationErrorResponse = (message) => {
+    return response({
+        message
+    }, 400);
+}
+
+module.exports.ProductExistsErrorResponse = (message) => {
+    return response({
+        message
+    }, 422);
```

Comment:
could be simplified -> status code can be as argument

Reply:
because fo exception is about unprocessible entity due to duplicate of product, so error code here is incapsulated to the exception. i.e. there should not be any case with other error code, only 422.

If you will have any questions or concerns feel free to contact me in the teams, thanks